gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue Apr 23 13:35:27 PDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
Also, run `./gradlew wrapper` and commit those changes.

Reply:
`./gradlew wrapper` run and new diff pushed up; merging once the build is green.

Reply:
Ah looks like I can't merge; ready for you to merge whenever @pyricau.

Reply:
Just invited you to the `Square Open Source Android` team. Welcome to Square!